Jenkinsfile

```diff
@@ -1,19 +1,27 @@
 node("docker") {
     cleanWs()
+    // Subtract repo name from the repo url (https://REPO_NAME/ -> REPO_NAME/)
+    withCredentials([string(credentialsId: 'repo21-url', variable: 'REPO21_URL')]) {
+        echo "${REPO21_URL}"
+        def repo21Name = "${REPO21_URL}".substring(8, "${REPO21_URL}".length())
+        env.REPO_NAME_21="$repo21Name"
+    }
     def architectures = [
-            [pkg: 'jfrog-cli-windows-amd64', goos: 'windows', goarch: 'amd64', fileExtention: '.exe', chocoImage: 'linuturk/mono-choco'],
-            [pkg: 'jfrog-cli-linux-386', goos: 'linux', goarch: '386', fileExtention: '', debianImage: 'i386/ubuntu:16.04', debianArch: 'i386'],
-            [pkg: 'jfrog-cli-linux-amd64', goos: 'linux', goarch: 'amd64', fileExtention: '', debianImage: 'ubuntu:16.04', debianArch: 'x86_64', rpmImage: 'tgagor/centos:stream8'],
-            [pkg: 'jfrog-cli-linux-arm64', goos: 'linux', goarch: 'arm64', fileExtention: ''],
-            [pkg: 'jfrog-cli-linux-arm', goos: 'linux', goarch: 'arm', fileExtention: ''],
-            [pkg: 'jfrog-cli-mac-386', goos: 'darwin', goarch: 'amd64', fileExtention: ''],
-            [pkg: 'jfrog-cli-linux-s390x', goos: 'linux', goarch: 's390x', fileExtention: ''],
-            [pkg: 'jfrog-cli-linux-ppc64', goos: 'linux', goarch: 'ppc64', fileExtention: ''],
-            [pkg: 'jfrog-cli-linux-ppc64le', goos: 'linux', goarch: 'ppc64le', fileExtention: '']
+            [pkg: 'jfrog-cli-windows-amd64', goos: 'windows', goarch: 'amd64', fileExtension: '.exe', chocoImage: 'linuturk/mono-choco'],
```

Comment:
Publishing to Choco was removed in this PR. I see [here](https://community.chocolatey.org/packages/jfrog-cli#versionhistory) that we continue to publish the v1 versions under the same package as v2 'jfrog' versions.
If we'd like to continue publishing v1 versions the functionality needs to be added to this PR, and if not then this `chocoImage` property can be removed.

Reply:
I removed the choco image property.